vector/src/main/res/values/strings.xml

```diff
@@ -1534,7 +1534,6 @@ Why choose Riot.im?
     <string name="settings_other_third_party_notices">Other third party notices</string>
     <string name="navigate_to_room_when_already_in_the_room">You are already viewing this room!</string>
 
-    <string name="quick_reactions">Quick Reactions</string>
```

Comment:
Note that it will come back because this file will be copied from Riot. We will cleanup strings after.